generator/src/main/kotlin/io/lsdconsulting/stub/writer/StubWriter.kt

```diff
@@ -22,55 +21,28 @@ class StubWriter(processingEnv: ProcessingEnvironment) {
         messager = processingEnv.messager
     }
 
-    fun writeStubBaseFile(controllerModel: ControllerModel) {
-        try {
-            val builderFile = processingEnv.filer.createSourceFile(controllerModel.stubBaseFullyQualifiedName)
-            val stubBasePathName = builderFile.toUri().path.replace(
-                "generated/source/kapt/main",
-                "generated-stub-sources"
-            )
-            val directory = stubBasePathName.replace("StubBase.tpl", "")
-            messager.printMessage(Diagnostic.Kind.NOTE, "Creating directory:$directory")
-            Files.createDirectories(Path.of(directory))
-            messager.printMessage(Diagnostic.Kind.NOTE, "Creating file:$stubBasePathName")
-            val path = Files.createFile(Path.of(stubBasePathName))
-            PrintWriter(builderFile.openWriter()).use { writer ->
-                stubBaseTemplate.evaluate(writer, mapOf("packageName" to controllerModel.packageName))
+    fun writeStubFile(model: Model) {
```

Comment:
I don't like how much is going on in this method - maybe start with some newlines to tell the story better, then possibly extract methods

Reply:
Refactored